www/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "disableSourceOfProjectReferenceRedirect": true,
+    "disableSolutionSearching": true,
+    "target": "esnext",
```

Comment:
Should we use `"target": "es6"` since ES6 is the minimum Ecmascript the platform supports currently. If we use `esnext` here, we're only supporting the absolute latest spec, which likely won't have great browser compatibility.

> The special ESNext value refers to the highest version your version of TypeScript supports. This setting should be used with caution, since it doesn’t mean the same thing between different TypeScript versions and can make upgrades less predictable.
([source](https://www.typescriptlang.org/tsconfig#target))

Reply:
Yes, it`s good idea
Corrected